src/app/templates/SeedPhraseInput/SeedWordInput.tsx

```diff
@@ -50,37 +50,50 @@ export const SeedWordInput: FC<SeedWordInputProps> = ({
     [value]
   );
 
+  const errorCaption = (submitted && !value) || isWordSpellingError;
+
+  useEffect(() => {
+    if (isWordSpellingError) {
+      setWordSpellingErrorsCount(prev => prev + 1);
```

Comment:
There may be no actual spelling error in this case because a user may still be typing. Please consider adding debounce for the increment